templates/nodes/dummy_node_params.yaml

```diff
@@ -0,0 +1,4 @@
+dummy_node:
+  ros__parameters:
+
+    perfect_number: "twentyeight"
```

Comment:
I'd choose `42`, but to each their own ;)

Reply:
Not really, 28 is one of the perfect numbers that is why it is chosen. And 23 is the address of the office in Karlsruhe :)